build.sbt

```
@@ -93,10 +93,12 @@ lazy val zioPrelude = "1.0.0-RC16"
 lazy val scala211 = "2.11.12"
 lazy val scala212 = "2.12.17"
 lazy val scala213 = "2.13.8"
+lazy val scala31 = "3.1.1"
```

pour ZIO-direct, il faut un peu plus

```suggestion
lazy val scala32 = "3.2.1"
```